fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseWriter.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Response writer interface.
```

Comment:
We should provide a better class level JavaDoc. Nothing here says anything that the class declaration doesn't already.

In general, when writing JavaDoc you should focus on the `what`, and possibly the `why` depending on context (and how much tolerance your co-developers have for large class docs ;). 

For example, I might write something like the following:
```
The interface for objects that write fully-processed ResultSets back to the user. This allows customers 
to fully customize how they choose to serialize the results from Fili based on the request.
```